data/human/free worlds 1 start.txt

```diff
@@ -0,0 +1,2228 @@
+			`	Alondo is silent. As you're about to leave the spaceport for the landing pads, you hear someone shout behind you. "No need to be a stranger, Gruyere. Can't stick around for a drink?" You turn to look at who it is, but Alondo keeps walking. You find Samuel Remington approaching, paying you no mind. "Are you still mad at me, Gruyere?" he asks`
+			choice
+				`	(Stand in Remington's way.)`
+					goto stand
+				`	(Don't engage.)`
+			
+			`	Remington stops a few paces past you as Alondo boards your ship. Remington shakes his head before looking back to you. "I guess a guy can't talk to an old friend."`
+				goto remington
+			
+			label stand
+			`	You take a step into Remington's path as Alondo boards your ship. Remington stops face-to-face with you. "What, a guy can't talk to an old friend?" he says while looking over your shoulder`
+			label remington
+			choice
+				`	"He's clearly not interested"`
+					goto interested
+				`	"You know Alondo?"`
+			
+			`	"More than most," Remington says with a chuckle. "He was never one to show gratitude for gifts. I hope you enjoy the new toys in those shipments I just secured more than he does."`
+				goto alondo
+			
+			label interested
+			`	"As if that's ever stopped me before," Remington says with a chuckle. "He was never one to show gratitude for gifts. I hope you enjoy the new toys in those shipments I just secured more than he does."`
+			
+			label alondo
+			`	"What new toys?" you ask.`
+			`	"You'll see," he says, turning back to the spaceport and leaving you.`
+			`	You join Alondo on your ship and find him solemnly staring out a window.`
+			choice
+				`	"It seems like the negotiations went pretty well."`
+					goto well
+				`	"What was that all about?"`
+					goto that
+				`	"What do we do about Katya?"`
+					goto katya
+			
+			label that
+			`	"Samuel and I have... a history. One I'd rather not discuss," Alondo responds. "Anyway, we must continue with our mission. Time for us to make a visit to Tarazed. Let's get going."`
+				accept
+			
+			label well
+			`	"They did. Although it feels like it was almost too easy." Alondo pauses for a moment. "Anyway, we must continue with our mission. Time for us to make a visit to Tarazed."`
+				accept
+			
+			label katya
+			`	"I'll have to talk that over with the Council when we get back. But if she is a prisoner of the Intelligence Bureau, there is no way we can rescue her. Her only hope is that they will recognize her innocence. For now, we must continue with our mission. Time for us to make a visit to Tarazed."`
```

Comment:
When you first when through this file, GitHub was marking absolutely every line as an addition (so you might actually be catching some errors from him rather than me). This specific part is from MZ.

Keep in mind too though that we don't actually learn that it was Navy Intel until the beginning of FWR.

Reply:
Okay, so what should we do with this line then?

Reply:
Nothing. Alondo doesn't have perfect information, so there's nothing wrong with this being off.